src/View/Helper/FlashHelper.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+declare(strict_types=1);
```

Comment:
@ADmad Shouldnt there be a newline between this and and namespace?

Reply:
Enforcing a newline after `declare()` would add a newline above the file level docblocks in core (which have the copyright and license tags).

I don't mind either way. The slevomat sniff could be configured accordingly.

Reply:
I still hate that those declare those additions cannot be added to open tag on file level.. :(
```
<?php declare(strict_types=1);
```
That resolves the issue as the newline then added below is trivial.

Reply:
https://github.com/rectorphp/rector/blob/master/src/CodingStyle/AfterRectorCodingStyle.php#L1 etc

Reply:
You are gonna "love" PSR12 https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#example.

I wants an empty line between `<?php` and `declare()` :slightly_smiling_face:.